2018/OMI-2018-Reforma/settings.json

```
@@ -18,11 +19,10 @@
   },
 
   "misc": {
-    "alias": "OMI2018-Reforma",
+    "alias": "OMI-2018-Reforma",
```

```suggestion
    "alias": "OMI2018-Reforma",
```

El alias correcto es este. Hay que checar cómo está en [omegaUp](https://omegaup.com/arena/problem/OMI2018-Reforma).